int32 id
geometry_msgs/Point p1
geometry_msgs/Point p2
geometry_msgs/Point p3
geometry_msgs/Point p4
geometry_msgs/Point start
geometry_msgs/Point end
Wall main_wall
int32 a12
int32 a23
int32 a34
int32 a41
